UEBPrep/app/controllers/sessions_controller.rb

```diff
@@ -4,9 +4,12 @@ def new
   end
 
   def create
-    users = User.where(email: params[:session][:email])
+    users = User.where(username: params[:session][:username])
     #If it's greater than one, we violated the unique email constraint
     #on the user model
+    if users.count < 1
+      users = User.where(email: params[:session][:username])
```

Comment:
@caesarfish  on second look, it appears the purpose of this catch is to allow someone to login with either their username or password. Line 7 first trys to find the user by username, and then if they don't find a user with that username, they look for someone who has the same email as the username entered. So, to answer my question, no we don't need to update this line.